cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,55 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import { SettingsPageObject } from '../support/pages/settings.pageObject';
+
+const settingsPage = new SettingsPageObject();
 describe('Settings page', () => {
+  let user;
   before(() => {
 
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
   beforeEach(() => {
 
+    cy.task('db:clear');
+    cy.visit('/');
+    cy.login(user.email, user.username, user.password);
+    settingsPage.visit();
   });
 
   it('should provide an ability to update username', () => {
 
+    settingsPage.typeUsername(user.username);
+    settingsPage.submitForm();
+    settingsPage.assertProfilePage(user.username);
```

Comment:
The assertion here checks if the profile page displays the updated username, which is good. Ensure similar assertions are present in other tests to verify changes.